pkgs/development/python-modules/crate/default.nix

```diff
@@ -46,7 +46,11 @@ buildPythonPackage rec {
     "test_dynamic_http_port"
     "test_environment_variables"
     "test_verbosity"
-  ];
+  ] ++ (assert lib.versionOlder version "0.35.3"; [
+    # the following tests require older Pandas
+    "test_bulk_save_dask"
+    "test_bulk_save_pandas"
+  ]);
```

Comment:
This assertion is a a bit wierd. Why not use `lib.optionals`? Plus, it'd be nice to put a comment that these can be removed if the correct dependencies will be fixed. Additionally, in anycase we can update the package to `0.35.2`? https://github.com/crate/crate-python/releases